### DISCUSSION TITLE
Using the in statement

### DISCUSSION THREAD

**Discussion by stacode123** [cc-tweaked/CC-Tweaked#1314] [Q&A] [answered]
IF i have a string a ="a" and a string b= "ab" can i use
if a in b then
something
end
?

**Comment by SquidDev** [comment, marked as answer]
No - the `in` keyword is only used for looping over things (`for k, v in pairs(some_table) do ... end`).

If you want to check if one string is in another, you can use Lua patterns and [`string.find`](https://www.lua.org/manual/5.1/manual.html#pdf-string.find):

```lua
local a = "a"
local b = "ab"
if b:find(a, 1, true) then
  -- something
end
```

**Reply by stacode123** [reply]
OK got it thanks
